Assets/_Scripts/Enemy/EnemyPathfinding.cs

```diff
@@ -59,6 +56,16 @@ public void SetState(PathfindingState state)
     // TODO: ZA - Estelle - DEBUG STATE CHANGER - REMOVE
     private void Update()
     {
+        // Detect Entrance -> Hover state change
+        if (_state == PathfindingState.Entrance)
+        {
+            BezierKnot endpoint = _splineContainer.Splines[_chosenPath].Last();
+            if (Vector2.Distance(transform.position, SplineLocalToWorldPoint(endpoint.Position)) < 0.5f)
+            {
+                SetState(PathfindingState.Hover);
+            }
+        }
+
         if (Input.GetKeyDown(KeyCode.A))
```

Comment:
If I remember correctly, we used this to test the entrance state.
I think we can remove this now?